### PR TITLE
Fixes #16149: Broken relay shared-files cleanup cron

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16149

### CODE REVIEW

File: relay/sources/Makefile [hunk 1]
```diff
@@ -135,6 +135,7 @@ install: build
 	install -m 644 rudder-relay-apache $(DESTDIR)/etc/sysconfig/rudder-relay-apache
 	install -m 644 rudder-relay.cron $(DESTDIR)/etc/cron.d/rudder-relay
 	install -m 644 rudder-relay.sudo $(DESTDIR)/etc/sudoers.d/rudder-relay
+	install -m 755 relay-cleanup.sh $(DESTDIR)/opt/rudder/bin/relay-cleanup
```
Comment (peckpeck) [inline relay/sources/Makefile]: why not remove the .sh in the source too ?

File: relay/sources/relay-cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# remove all files and their metadata in BASEDIR that have expired
+
+BASEDIR="/var/rudder/shared-files"
+date=$(date +%s)
+find "${BASEDIR}" -type f -name '*.metadata' | xargs grep -H 'expires=' | sed 's/^\(.*\).metadata:expires=/\1 /' |
```
Comment (peckpeck) [inline relay/sources/relay-cleanup.sh]: is the expiration an integer timestamp ?

**Comment by amousset** [comment]
PR updated with a new commit

File: relay/sources/relay-cleanup.sh (hunk 2 above)
Comment (amousset) [reply relay/sources/relay-cleanup.sh]: This is the previous cronjob without modification. (But yes it is a timestamp IIRC)

File: relay/sources/Makefile (hunk 1 above)
Comment (amousset) [reply relay/sources/Makefile]: done

**Pull request merged**